Docs/Configuration.tex

```diff
@@ -1666,25 +1649,25 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
   performing early boot identity mapping of assigned virtual addresses to physical
   memory.
 
-  \emph{Note}: The necessity of this quirk is determined by early boot failures. Currently,
-  new firmware with memory protection support (such as OVMF) do not support this quirk. See
-  \href{https://github.com/acidanthera/bugtracker/issues/719}{acidanthera/bugtracker\#719}.
+  \emph{Note}: The need for this quirk is determined by early boot failures. However, new
+  firmware with memory protection support (such as OVMF) do not currently support this quirk.
+  See \href{https://github.com/acidanthera/bugtracker/issues/719}{acidanthera/bugtracker\#719}.
 
 \item
   \texttt{SignalAppleOS}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
   \textbf{Description}: Report macOS being loaded through OS Info for any OS.
 
-  This quirk is useful on Mac firmware, which behaves differently in different OS.
+  This quirk is useful on Mac firmware, which behaves differently for different OS.
```

Comment:
Prefer original.

Reply:
Get you but a toss up.

Issue is whether "Mac firmware" is ever "in" a different OS. However a different OS might run on Mac firmware.
Needs some feedback from Vit on actual intent

Reply:
I was (I guess, on being more explicit about it since you are rightly being so too!) reading "in" as short for "in the context of"... which I think would make it fine. And would also mean that "OS" should be pluralised - I guess as "OSes" - which is something I thought of when I first read this, but didn't mention, as that looks pretty horrible and - I have just now checked - is not done anywhere else in the docs.

Reply:
cc @vit9696

Reply:
which loads different operating systems with different hardware configurations.